solutions/src/main/java/ru/sberbank/school/task08/SerializableManager.java

```diff
@@ -17,17 +18,36 @@ public SerializableManager(@NonNull String filesDirectoryPath) {
 
     @Override
     public void initialize() {
-        throw new UnsupportedOperationException("Implement me!");
-    }
+//        throw new UnsupportedOperationException("Implement me!");
+        }
```

Comment:
&#39;method def rcurly&#39; has incorrect indentation level 8, expected level should be 4.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-806899601>*